guideline-17.md

```diff
@@ -1,7 +1,7 @@
 <h4>17. Plugins must respect trademarks, copyrights, and project names.</h4>
 
-The use of trademarks or other projects as the sole or initial term of a plugin slug is prohibited unless proof of legal ownership/representation can be confirmed. For example, the <a href="http://wordpressfoundation.org/trademark-policy/">WordPress Foundation has trademarked the term “WordPress”</a> and it is a violation to use “wordpress” in a domain name. This policy extends to plugin slugs, and we will not permit a slug to begin with another product's term.
+The use of trademarks or other projects as the sole or initial term of a plugin slug is prohibited unless proof of legal ownership/representation can be confirmed. For example, the <a href="http://wordpressfoundation.org/trademark-policy/">WordPress Foundation has trademarked the term “WordPress”</a> and it is a violation to use “wordpress” in a domain name. This policy extends to plugin slugs, and we will not permit a slug or display name to begin with another product's term.
```

Comment:
Recommend to remove “we” statement from this.

“and slugs or display names beginning with another product’s term are prohibited/not allowed.”